tests/e2e/framework/utils.go

```diff
@@ -384,6 +386,139 @@ func (f *Framework) AssertMetricsEndpointUsesHTTPVersion(endpoint, version strin
 	return nil
 }
 
+// createServiceAccount creates a service account
+func (f *Framework) SetupRBACForMetricsTest() error {
+	ocPath, err := exec.LookPath("oc")
```

Comment:
We do this a few times in this patch, and there are other places where we're performing the same lookup.

We might consider caching the path to this binary in the framework struct somewhere.